great_expectations/expectations/expectation.py

```diff
@@ -1367,6 +1367,188 @@ def _validate(
         )
 
 
+class MulticolumnMapExpectation(TableExpectation, ABC):
```

Comment:
Please revise capitalization per: result of slack convo inside the team

Reply:
@jcampbell I cannot tell with certainty what the consensus in Slack is for the naming.  It seems that the Team prefer `MultiColumn` -- no problem.  How about we update this uniformly across the expectations and metrics as quick follow-on PR?  If agree, I will add it to JIRA and take care of it right away.  Thanks.

Reply:
Sounds good.

Reply:
This is JIRA `https://superconductive.atlassian.net/browse/GE-442`.